lib/takeaway.rb

```diff
@@ -0,0 +1,34 @@
+class Takeaway
+
+  MENU = {
```

Comment:
Good job making the menu a constant, this tripped me up a bit when I was coding!